src/main/resources/shared.yaml

```diff
@@ -66,6 +66,9 @@ definitions:
       pending:
         description: Flag indicating whether the user is pending, i.e. registered but not activated yet.
         type: boolean
+      stale:
+        description: Flag indicating whether the user is a stale user, i.e. is no longer in idm/am but moved to stale users table.
```

Comment:
lets not mention forgerock here

Reply:
@nikola-naydenov-hmcts removed forgerock mention